content/en/blog/news/2022-introduction-to-sbom.md

```diff
@@ -0,0 +1,72 @@
+A more detailed explanation about those types of attacks is [here](https://slsa.dev/spec/v0.1/threats)
+
+Due to its importance and being a critical issue, generating SBOM for your software adds another layer of protection to this threat.
+
+## Definition: What is SBOM?
+
+As far as we know, developers around the world are building web applications using hundreds of third-party open-source libraries and packages. You can confidently tell that 90% of the software products around the world are built over open-source components. With that in mind, we need to keep track of using these dependencies while building our applications. What if there are vulnerabilities in the libraries we use? How to efficiently protect ourselves against it?.
+
+**[Software Bill Of Materials](https://en.wikipedia.org/wiki/Software_supply_chain#:~:text=Software%20vendors%20often,could%20harm%20them.)** (SBOM) is a complete formally structured list of the materials (components, packages, libraries, SDK) used to build (i.e. compile, link) a given piece of software and the supply chain relationships between all these materials.
+
+It is an inventory of all the components developers used to make this software. It has many formats and many generating tools but all have the same purpose in the end.
+***Example: a simple formatted SBOM of Ubuntu alpine docker image using [syft](https://anchore.com/sbom/how-to-generate-an-sbom-with-free-open-source-tools/)***
+
+```
+✔ Loaded image  
+ ✔ Parsed image  
+ ✔ Cataloged packages      [14 packages]
+NAME                    VERSION      TYPE 
+alpine-baselayout       3.2.0-r18    apk   
+alpine-keys             2.4-r1       apk   
+apk-tools               2.12.7-r3    apk   
+busybox                 1.34.1-r3    apk   
+ca-certificates-bundle  20191127-r7  apk   
+libc-utils              0.7.2-r3     apk   
+libcrypto1.1            1.1.1l-r7    apk   
+libretls                3.3.4-r2     apk   
+libssl1.1               1.1.1l-r7    apk   
+musl                    1.2.2-r7     apk   
+musl-utils              1.2.2-r7     apk   
+scanelf                 1.3.3-r0     apk   
+ssl_client              1.34.1-r3    apk   
+zlib                    1.2.11-r3    apk
+
+```
+
+Here it shows only softwares included in the final layer of the container (default choice by syft). If we want to view a detailed SBOM with one detailed format, we can run `syft alpine -o spdx-json`. This will view the output as `.json` file following the `spdx` format (will discuss that later)
+
+## Use cases in Supply chain security
+
+What makes a supply chain attack susceptible is the lack of transparency and visibility about whether the software gets affected by a recent exploit or not. This greatly affects both developers and users of the product.
```

Comment:
may be use `producers` instead of `developers` and `consumers` instead of `users`?

Reply:
Done